hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/DistributedScheduledExecutorService.java

```diff
@@ -91,6 +98,10 @@ public Object createNew(String name) {
         }
     };
 
+
+    private final ConcurrentHashMap<String, LocalScheduledExecutorStatsImpl> statsMap
```

Comment:
Need to clear this during 'shutdown' method.

Reply:
Why do we declare the 'statsMap' in the service? It seem counter intuitive to handle the state here. The only benefit is that we know the size of the map during cloning (ie. `getStats()`), other than that, I guess its Container can have its own stats that can be collected here.

Reply:
Because the stats map is exposes here through the statistics aware service.